exercises/ocr-numbers/tests/ocr-numbers.rs

```diff
@@ -0,0 +1,212 @@
+extern crate ocr_numbers as ocr;
+
+// use ocr_numbers::*;
+
+#[test]
+#[cfg_attr(rustfmt, rustfmt_skip)]
+fn recognizes_0() {
+    let input = " _ \n".to_string() +
+                "| |\n" +
+                "|_|\n" +
+                "   ";
+
+    assert_eq!("0", ocr::convert(input).unwrap());
+}
+
+#[test]
+#[ignore]
+#[cfg_attr(rustfmt, rustfmt_skip)]
+fn recognizes_1() {
+    let input = "   \n".to_string() +
+                "  |\n" +
+                "  |\n" +
+                "   ";
+
+    assert_eq!("1", ocr::convert(input).unwrap());
+}
+
+#[test]
+#[ignore]
+#[cfg_attr(rustfmt, rustfmt_skip)]
+fn unrecognized_chararcters_return_question_mark() {
+    let input = "   \n".to_string() +
+                "  _\n" +
+                "  |\n" +
+                "   ";
+
+    assert_eq!("?", ocr::convert(input).unwrap());
+}
+
+#[test]
+#[ignore]
+#[cfg_attr(rustfmt, rustfmt_skip)]
+fn input_with_lines_not_multiple_of_four_panics() {
```

Comment:
well these aren't really panicking

Since I assume we *do* want to keep the current behavior of error (rather than panic) I assume that means just to change the test name